test/test_error.c

```diff
@@ -5,19 +5,51 @@ Written by Mark C. Miller, miller86@llnl.gov
 LLNL-CODE-707197. All rights reserved.
 
 This file is part of H5Z-ZFP. Please also read the BSD license
-https://raw.githubusercontent.com/LLNL/H5Z-ZFP/master/LICENSE 
+https://raw.githubusercontent.com/LLNL/H5Z-ZFP/master/LICENSE
 */
 
+#if !defined(_WIN32) && !defined(_GNU_SOURCE)
 #define _GNU_SOURCE /* ahead of ALL headers to take proper effect */
+#endif
 #include <assert.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <strings.h>
+#if defined(_WIN32)
+#define _USE_MATH_DEFINES
+#include <math.h>
+#include <io.h>
+#define strncasecmp _strnicmp
+
+#include <ctype.h>
+char * strcasestr(s, find)
```

Comment:
Not sure if it applies here but In VisIt, I see `strcasestr` for Windows being handled as...

```
#ifdef _MSC_VER
    #include<shlwapi.h>
    StrStrIA(variableNames[0].c_str(), "coord"))
#else
    strcasestr(variableNames[0].c_str(), "coord"))
#endif
```

Reply:
See PR #112